src/osd.c

```diff
@@ -223,173 +223,183 @@ preview_cycled_view(struct view *view)
 	wlr_scene_node_raise_to_top(osd_state->preview_node);
 }
 
-void
-osd_update(struct server *server)
+static void
+render_osd(cairo_t *cairo, int w, int h, struct wl_list *node_list,
+	struct view *cycle_view, struct theme *theme, bool show_workspace,
+	const char *workspace_name)
```

Comment:
Style nit: In the rest of the codebase we are using two tab indentation for continued argument lines. E.g.
```c
static void
render_osd(cairo_t *cairo, int w, int h, struct wl_list *node_list,
		struct view *cycle_view, struct theme *theme, bool show_workspace,
		const char *workspace_name)
{
```

Reply:
Will fix.